cves/kernel/CVE-2017-18255.yml

```diff
@@ -305,9 +303,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: true
+  any_discussion: true
+  note: There was an extensive conversation between the developers within this commit where the person who made the commit discovered it via a fuzzer and was discussing with fellow developers of how to fix this issue and what type of error it is in order to patch it
```

Comment:
A little bit of an overly long sentence. Maybe cut it up into two for readability.